git-refresher/syncing-changes-to-remote.qmd

```diff
@@ -1,18 +1,55 @@
---- 
+---
 title: "Syncing to the remote"
 ---
 
-...
+This section will guide you through syncing your local repo to the remote one.
+
+Any change made so far is only being applied to the local copy of the repository (i.e. the copy on your laptop). To apply your changes to the remote repository (i.e. on GitHub or Dev Ops), you need to “push” the changes.
```

Comment:
GitHub is a flavour of DevOps, so saying "GitHub or DevOps" will get a strange look from anyone who happens to be a bit more familiar with what DevOps is. Doesn't matter so much for the level of user we're aiming at here, but worth changing this to "GitHub or Azure DevOps".